src/parser/meta.rs

```diff
@@ -375,7 +379,7 @@ fn map_meta_flag(
 }
 
 fn construct_value_from_meta_values(
-    meta_values_array: Vec<(u8, &[u8])>,
+    meta_values_array: Vec<(u8, Option<&[u8]>)>,
```

Comment:
Consequence of the parsing change, the tokens that correspond to the first char of each flag are now optional